README.md

```diff
@@ -10,4 +10,4 @@ Describe how you approached to problem, and what tools and techniques you used t
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://beepage.netlify.app/
```

Comment:
To fix it you need to create `_redirects` file in the `public` directory of your react project and paste it inside:
``` 
/* /index.html 200
```


![image](https://user-images.githubusercontent.com/50926159/120888177-2b5c5f00-c5f7-11eb-959a-c413b08ef7e2.png)

Reply:
We already have that is our public folder?